documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,262 @@
+
+You can configure your listener to use TLS or SCRAM-SHA authentication.
+If you are using an authorization server, you can use token-based link:{BookURLUsing}#assembly-oauth-authentication_str[{oauth} authentication] and link:{BookURLUsing}#assembly-oauth-authorization_str[{oauth} authorization].
+
+When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent Kafka configuration:
+
+* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
+* `KafkaUser.spec.authorization` matches `Kafka.spec.kafka.authorization`
+
+NOTE: Authentication between Kafka users and Kafka brokers depends on the authentication settings for each.
+For example, it is not possible to authenticate a user with TLS if it is not also enabled in the Kafka configuration.
+
+Strimzi operators automate the configuration process:
+
+* The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
+* The User Operator creates the user representing the client and sets up the user CA certificates for secure access to the Kafka cluster.
+
+In this procedure, the certificates generated by the Cluster Operator are used, but you can replace them by link:{BookURLUsing}#installing-your-own-ca-certificates-str[installing your own certificates].
+You can also configure your listener to link:{BookURLUsing}#kafka-listener-certificates-str[use a Kafka listener certificate managed by an external Certificate Authority].
+
+Certificates are available in PKCS #12 format (.p12) and PEM (.crt) formats.
+
+.Prerequisites
+
+* The Kafka cluster is available for the client
+* The Cluster Operator and User Operator are running in the cluster
+* A client outside the Kubernetes cluster to connect to the Kafka cluster
+
+.Procedure
+
+. Configure the Kafka cluster with an `external` Kafka listener.
++
+* Define the authentication required to access the Kafka broker through the listener
+* Enable authorization on the Kafka broker
++
+For example:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+spec:
+  kafka:
+    # ...
+    listeners: <1>
+      external:
+        type: _LISTENER-TYPE_ <2>
+        tls: true <3>
+        authentication:
+          type: tls <4>
+        configuration:
+          preferredAddressType: InternalDNS <5>
+        networkPolicyPeers: <6>
+          - podSelector:
+              matchLabels:
+                app: kafka-consumer
+          - namespaceSelector:
+              matchLabels:
+                project: my-project
+        overrides: <7>
+          bootstrap:
+            nodePort: 32100
+          brokers:
+            - broker: 0
+              nodePort: 32000
+            - broker: 1
+              nodePort: 32001
+            - broker: 2
+              nodePort: 32002
```

Comment:
I think this is confusing and should be removed. These options apply only in some situations so I think they will confuse the read rather than help. You should leave them out and just link to the using guide or to the APi reference.

Reply:
Okay. I'll drop the example with `# ...` and leave the callout at the overrides property.